notebooks/PFT-Gridding.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>Based on our telecon, I think you actually want something more succinct: </p><p> </p><pre class="ql-syntax" spellcheck="false">cropwtsmask = cropwts&lt;0.005
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/ctsm_py/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>